app/core/api/classrooms.js

```diff
@@ -182,4 +182,11 @@ module.exports = {
       json: { courseId }
     }))
   },
+
+  fetchNamesForUser (ids) {
```

Comment:
this sounds to me that it should be a user API instead of classroom API. Like get all classrooms of the user so it should better fit in the user space